js/utils/start-page.js

```diff
@@ -0,0 +1,23 @@
+const adsForm = document.querySelector('.ad-form');
+const mapFilter = document.querySelector('.map__filters');
+
+const disablePage = () => {
+  adsForm.classList.add('ad-form--disabled');
+  mapFilter.classList.add('ad-form--disabled');
+
+  [...adsForm.elements].forEach((element) => element.setAttribute('disabled', ''));
+  [...mapFilter.elements].forEach((element) => element.setAttribute('disabled', ''));
+};
```

Comment:
```suggestion
const togglePage = (isActive) => {
  adsForm.classList.toggle('ad-form--disabled', isActive);
  mapFilter.classList.toggle('ad-form--disabled', isActive);

  [...adsForm.elements].forEach((element) => element.disable = isActive);
  [...mapFilter.elements].forEach((element) => element.disable = isActive);
};
```
можешь все в одной функции сделать